BaseCollector.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from abc import ABC, abstractmethod
+from os import getenv
+
+import modules.TimedBlacklist as blacklist
+from modules.NetboxHelper import NetboxHelper
+from modules.VCenterConnection import VCenterConnection
+
+logger = logging.getLogger('esxi-exporter')
+
+
+class BaseCollector(ABC):
+
+    def __init__(self) -> None:
+        super().__init__()

```

Comment:
What's the purpose of this? 
FYI: Initialisation of the ABC class is done here already: https://github.com/python/cpython/blob/master/Lib/abc.py#L84-L87